libpod/container_commit.go

```diff
@@ -162,7 +162,7 @@ func (c *Container) Commit(ctx context.Context, destImage string, options Contai
 			importBuilder.SetWorkDir(splitChange[1])
 		}
 	}
-	candidates, _, err := util.ResolveName(destImage, "", sc, c.runtime.store)
```

Comment:
@rhatdan this popped up as part of the vendoring, I'm pretty sure this change is correct.  Can you verify please?

Reply:
yup

Reply:
That extra field is for transport,